Tests/UnitTests/MessageBirdUnitTests/MockRestClient.cs

```diff
@@ -104,10 +109,11 @@ public MockRestClient AndReturns(string responseBody)
         /// <summary>
         /// Sets the expected (HTTP) method.
         /// </summary>
-        public MockRestClient FromEndpoint(string method, string uri)
+        public MockRestClient FromEndpoint(string method, string uri, string endpoint = null)
```

Comment:
Is there any advantage to using a default value over utilising method overloading?

Reply:
Less duplicate code, so easier maintenance